azurerm/internal/services/web/app_service_environment_resource.go

```diff
@@ -66,6 +70,7 @@ func resourceArmAppServiceEnvironment() *schema.Resource {
 					string(web.InternalLoadBalancingModeNone),
 					string(web.InternalLoadBalancingModePublishing),
 					string(web.InternalLoadBalancingModeWeb),
+					string(InternalLoadBalancingModeWebPublishing),
 				}, false),
 			},
```

Comment:
This schema item should also be `ForceNew` as it cannot be changed after deployment, could you update that too while we're here?

Reply:
Makes sense, will fix it.